### PR TITLE
Bump from django 4.2.11 to 4.2.16

### PR DESCRIPTION
**Pull request by SmithPeder**
I see now that this is actually a duplicate of https://github.com/kolonialno/django-api-decorator/pull/23 🙃 But, I would prefer to change the `pyproject.yaml` as well, as the `>=3` semver is a bit broad. So, from the lockfile, we're running 4.2.11.

Fixes the 9 vulnerabilities here
``` sh
• django  installed 4.2.11  affected <4.2.16  CVE CVE-2024-45230
• django  installed 4.2.11  affected <4.2.16  CVE CVE-2024-45231
• django  installed 4.2.11  affected <4.2.14  CVE CVE-2024-39614
• django  installed 4.2.11  affected <4.2.14  CVE CVE-2024-39330
• django  installed 4.2.11  affected <4.2.14  CVE CVE-2024-39329
• django  installed 4.2.11  affected <4.2.14  CVE CVE-2024-38875
• django  installed 4.2.11  affected <4.2.15  CVE CVE-2024-41991
• django  installed 4.2.11  affected <4.2.15  CVE CVE-2024-41990
• django  installed 4.2.11  affected <4.2.15  CVE CVE-2024-42005
  ```

Closes #23

### CODE REVIEW

**Comment by SmithPeder** [comment]
I am not sure how we normally merge stuff in the decorator, as there are many projects/teams that use it 🤔

File: pyproject.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ packages = [{include = "django_api_decorator"}]
 
 [tool.poetry.dependencies]
 python = "^3.11"
-Django = ">=3"
+Django = "^4.2.16"
```
Comment (mortenkrane) [inline pyproject.toml:14]: I'm all for not support v. 3, and bumping it. But I think it makes sense to have a pretty broad acceptance for a library like this. I.e., `^4.2.16` doesn't allow Django 5, which I kinda think it should.

Comment (SmithPeder) [reply pyproject.toml:14]: Ahh, right, that's how it works 😬 Forgot about that 🤔

**Comment by mortenkrane** [comment]
> I am not sure how we normally merge stuff in the decorator, as there are many projects/teams that use it 🤔

Looks like you need to add a separate version bump commit like this: https://github.com/kolonialno/django-api-decorator/commit/a6709797583744967e14161c7e3407129fb84aaa, and then create a new release here: https://github.com/kolonialno/django-api-decorator/releases. From there the build, etc should be automatic.

**Review by mortenkrane — APPROVED** [review]
👍

**Pull request merged**